components/dash-core-components/src/components/Input.react.js

```diff
@@ -121,6 +131,23 @@ export default class Input extends PureComponent {
         } else {
             this.props.setProps({value});
         }
+        this.setState({pendingEvent: undefined});
+    }
+
+    debounceEvent(time = 0.5) {
+        const {value} = this.input.current;
+        const MILLISECONDS = 1000;
+        time = time * MILLISECONDS;
```

Comment:
Better not to modify a var in place if we don't need to, just multiply it in the `setTimeout` call. And to your other question, you're welcome to add `1000` to the [already long list of exclusions in the linter](https://github.com/plotly/dash/blob/a7a12d180e16eac0a84b88e2b8dc8f7c7601cbaf/components/dash-core-components/.eslintrc#L120), I agree that it's a little silly to create a const just to satisfy the linter. But this would be a little more self-documenting if the var were called `seconds` or some such instead of `time`.